b3lb/rest/migrations/0008_add_record_models.py

```diff
@@ -0,0 +1,81 @@
+        ),
+        migrations.CreateModel(
+            name='RecordSet',
+            fields=[
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False, unique=True)),
+                ('meeting_id', models.CharField(max_length=100)),
+                ('created_at', models.DateTimeField(default=django.utils.timezone.now)),
+                ('record_ready_url', models.URLField(default='')),
```

Comment:
Adapt the BBB API naming `meta_bbb-recording-ready-url` ?

s/record_/recording_/

Reply:
changed naming.